src/model/profile.ts

```diff
@@ -0,0 +1,23 @@
+import mongoose, { Schema, Document } from 'mongoose';
+import { v4 as uuidv4 } from 'uuid';
+
+/**
+ * @exports
+ * @interface IDocumentProfile
+ * @extends {Document}
+ */
+export interface IDocumentProfile extends Document {
```

Comment:
We miss `IProfile` for code representative, we might need it in the response and interactive between models.